nautobot/core/views/generic.py

```diff
@@ -1278,18 +1278,53 @@ def get_required_permission(self):
     def get(self, request):
         return redirect(self.get_return_url(request))
 
+    def handle_bulk_delete_all(self, request, **kwargs):
+        model = self.queryset.model
+        self.template_name = "generic/object_bulk_destroy_all.html"
```

Comment:
Because we have views that currently override the base `object_bulk_delete.html`, I'd prefer that we avoid creating a separate template for this code branch - would be better to have conditional logic in the template itself, possibly passing in a new context variable as needed.